README.md

```diff
@@ -9,19 +9,25 @@ CCExtractor is a tool that produces subtitles from TV use. Global accessibility
 
 The official repository is ([CCExtractor/ccextractor](https://github.com/CCExtractor/ccextractor)) and master being the most stable branch. 
 
+This tool is written mainly on C.
```

Comment:
Shift and add this to the line above (L10). Also change it to "It is primarily written in C.".